app/assets/stylesheets/application.css.scss

```diff
@@ -15,64 +15,69 @@
  *= require foundation_and_overrides
 
  */
+ body, h2, h4 {
+   background-color: white;
+   font-family: 'Karla', sans-serif;
+ }
 
-body {
-  background-color: white;
-}
+ .review_body {
+   max-width: 500px;
+   word-wrap: break-word;
+ }
 
-.name {
-  padding-top: 7px;
-  padding-right: 10px;
-}
+ .top-bar {
+   background-color: #FCD720;
+ }
 
-.review_body {
-  max-width: 500px;
-  word-wrap: break-word;
-}
-
-.top-bar {
-  background-color: #7EBAC2;
-}
+ .top-bar-section li:not(.has-form) a:not(.button) {
+   background: #5CCEEE;
+ }
 
+ .upvote {
+   color: #E30922;
+ }
 
-.top-bar-section li:not(.has-form) a:not(.button) {
-  background: #E29714;
-}
-
-.upvote {
-  color: #E30922;
-}
-
-.downvote {
-  color: #E30922;
-}
+ .downvote {
+   color: #E30922;
```

Comment:
Color literals like `#E30922` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#E30922` should be written as `#e30922`